tests/State/HubTest.php

```diff
@@ -135,6 +135,30 @@ public function testPopScope(): void
     }
 
     public function testWithScope(): void
+    {
+        $scope = new Scope();
+        $hub = new Hub($this->createMock(ClientInterface::class), $scope);
+
+        $callbackReturn = $hub->withScope(function (Scope $scopeArg) use ($scope): string {
+            $this->assertNotSame($scope, $scopeArg);
+
+            return 'foobarbaz';
+        });
+
+        $this->assertSame('foobarbaz', $callbackReturn);
+
+        $callbackInvoked = false;
+
+        $hub->configureScope(function (Scope $scopeArg) use (&$callbackInvoked, $scope): void {
```

Comment:
Why are we testing `configureScope()` here? It looks like this code is duplicate from `testConfigureScope()`

Reply:
@ste93cry Yeah, I thought so too, but most test cases have this. If I understood correctly, this checks that the scope gets popped.

Reply:
Right, I didn't think about it